tests/test-meta_box_html.php

```diff
@@ -45,13 +46,16 @@ function testDrawWithEmptyFieldExpectsWPErrorReturned() {
      */
 	function testDrawWithFieldTypeFormsetCallsDrawFormset() {
 		//arrange
-		$field = array( 'type' => 'formset' );
+        $TestValidBox = new \TestValidBox;
```

Comment:
Make sure we're sticking to tabs for indentation in this plugin. (It's WordPress official PHP style.) Applies to many lines in this PR.